docs/site/clojure-lsp.md

```diff
@@ -16,19 +16,19 @@ By default you won't need to install/setup anything as Calva handles that for yo
 
 Calva is able to automatically start the clojure-lsp server for you and can be configured to start the server under various different conditions. These behaviours can be configured through the `calva.enableClojureLspOnStart` setting, which takes the following options:
 
-+ "when-workspace-opened"
-+ "when-file-opened"
++ "when-workspace-opened->workspace-root"
++ "when-file-opened->furthest-project"
 + "never"
 
-#### "when-workspace-opened" [default]
+#### "when-workspace-opened->workspace-root" [default]
 
-When set to `"when-workspace-opened"` Calva will start the clojure-lsp in the root of all opened vscode workspaces. All Clojure files in a workspace will be serviced by the clojure-lsp server running in that workspace. This behavior requires that you are opening workspaces with a valid Clojure project in the root (the directory must contain a `deps.edn`, `project.clj` or `shadow-cljs.edn` file).
+When set to `"when-workspace-opened->workspace-root"` Calva will start the clojure-lsp in the root of all opened vscode workspaces. All Clojure files in a workspace will be serviced by the clojure-lsp server running in that workspace. This behavior requires that you are opening workspaces with a valid Clojure project in the root (the directory must contain a `deps.edn`, `project.clj` or `shadow-cljs.edn` file).
 
 This is the default auto-start behaviour.
 
-#### "when-file-opened"
+#### "when-file-opened->furthest-project"
 
-When set to `"when-file-opened"` Calva will attempt to start the clojure-lsp server whenever a Clojure file is opened. The LSP server will be started in the outermost valid Clojure project or will fall back to starting in the workspace root if no valid Clojure project can be found. A directory is considered a Clojure project if it contains typical Clojure project files such as a `deps.edn`, `project.clj`, or `shadow-cljs.edn` file. When working in a mono-repo style project or in a multi-workspace VS Code configuration you may have multiple LSP servers running for each independent Clojure project root.
+When set to `"when-file-opened->furthest-project"` Calva will attempt to start the clojure-lsp server whenever a Clojure file is opened. The LSP server will be started in the outermost valid Clojure project or will fall back to starting in the workspace root if no valid Clojure project can be found. A directory is considered a Clojure project if it contains typical Clojure project files such as a `deps.edn`, `project.clj`, or `shadow-cljs.edn` file. When working in a mono-repo style project or in a multi-workspace VS Code configuration you may have multiple LSP servers running for each independent Clojure project root.
```

Comment:
> When working in a mono-repo style project or in a multi-workspace VS Code configuration you may have multiple LSP servers running for each independent Clojure project root.

Does this mean multiple LSP servers can be running for _the same_ independent project root? Or are you meaning to say that multiple LSP servers can be running, one for each independent Clojure project root? I think it reads like the former, but if it's the latter, can we make it a bit clearer?

Reply:
> I think it reads like the former, but if it's the latter, can we make it a bit clearer?

Yea it's the latter - I reworded it slightly, it should be more clear now.